notation/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { install } from './install';
+import { sign } from './sign'
+import { verify } from './verify';
+
+async function run() {
+    try {
+        let command = taskLib.getInput('command', true);
+        switch (command) {
+            case 'install':
+                await checkAndInstall();
+                break;
+            case 'sign':
+                await checkAndInstall();
+                await sign();
+                break;
+            case 'verify':
+                await checkAndInstall();
+                await verify();
+                break;
+            default:
+                throw new Error(`Unknown command: ${command}`);
```

Comment:
Yes, it is case sensitive because the value is defined in `task.json` and the type of `command` is a `pickList`
```
        {
            "name": "command",
            "type": "pickList",
            "label": "Command to run",
            "defaultValue": "install",
            "helpMarkDown": "example: install",
            "required": true,
            "options": {
                "install": "Install",
                "sign": "Sign",
                "verify": "Verify"
            },
            "groupName": "basicConfig"
        },
```